dir.props

```diff
@@ -287,9 +287,17 @@
   </PropertyGroup>
   
   <PropertyGroup>
+    <CombinedInstallerStart>$(PackagesOutDir)dotnet-runtime-</CombinedInstallerStart>
     <SharedHostInstallerStart>$(PackagesOutDir)dotnet-host-</SharedHostInstallerStart>
     <HostFxrInstallerStart>$(PackagesOutDir)dotnet-hostfxr-</HostFxrInstallerStart>
-    <SharedFrameworkInstallerStart>$(PackagesOutDir)dotnet-sharedframework-</SharedFrameworkInstallerStart>
+    <SharedFrameworkInstallerStart>$(PackagesOutDir)dotnet-runtime-</SharedFrameworkInstallerStart>
+
+    <!-- OSX specific intermediate package suffix -->
```

Comment:
I'm not sure I understand what this is for/why it is needed.  Can this comment explain why this is doing what it is doing?